packages/build-info/src/frameworks/next.ts

```diff
@@ -33,7 +33,7 @@ export class Next extends BaseFramework implements Framework {
     if (this.detected) {
       const nodeVersion = await this.project.getCurrentNodeVersion()
       if (nodeVersion && gte(nodeVersion, '10.13.0')) {
-        this.plugins.push('@netlify/plugin-nextjs')
+        this.plugins.push('@opennextjs/netlify')
```

Comment:
Should this use new package name for `next@>=13.5.1` only and otherwise use old version?